src/api/handlers/testimonial.py

```diff
@@ -102,7 +104,12 @@ def submit(self, request):
 
     # user submitted testimonial, so notify the community admins
     user_submitted = True
-    testimonial_info, err = self.service.create_testimonial(context, args, user_submitted)
+    is_edit = args.pop("testimonial_id", None)
```

Comment:
this does not look like it will work.  testimonials_update requires 'id' in args, which would be the same as 'testimonial_id'.

Reply:
the `id` is added to `args` on the frontend.  I am passing testimonial_id as a way of telling the backend this request is an update request.

Reply:
It looks like frontend-portal passes in id, and frontend-admin passes in testimonial_id.   In my opinion, both should pass in 'id' and if that integer corresponds to a testimonial that would be an update request